src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -3099,6 +3099,12 @@ public sealed class SocketsHttpHandlerTest_ConnectCallback_Http2 : SocketsHttpHa
         [InlineData(false)]
         public async Task ConnectCallback_UseNamedPipe_Success(bool useSsl)
         {
+            if (PlatformDetection.IsWindows && PlatformDetection.IsNativeAot)
```

Comment:
Why not an attribute?

Reply:
or it is not possible to have an "&&" check in the attribute?

Reply:
It's not.
The alternative we've been using is to either add the check into the method with a comment like I did or to introduce a new property that does the AND.

Personally I find the in-method approach cleaner if you're not reusing the same aggregate condition across multiple tests.

Reply:
> or it is not possible to have an "&&"-meaning check in the attribute?

You can use multiple PlatformDetection conditions with the ActiveIssue and other similar attributes. xunit runner will && all of them. 

This should do the right thing here:

`[ActiveIssue("https://github.com/dotnet/runtime/issues/73772", typeof(PlatformDetection), nameof(PlatformDetection.IsWindows), nameof(PlatformDetection.IsNativeAot))]`

Reply:
Thanks Jan, that's good to know. Changed it to use the Attribute instead.